ARCs/arc-0034.md

```diff
@@ -0,0 +1,63 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The purpose of this ARC is to set the parameters for proposals submitted by the community for evaluation by xGovs prior to a general governance vote for final approval.
+ 
+The Foundation’s goal is to ensure that the xGov process is defined by, and executed by, the Algorand community.
+
+The xGov program will cover evaluation of proposals regarding grants to fund early stage projects and other community input contributions.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+Being able to put measures for a vote is a significant upgrade to the power of the governors. Governors with this power **MUST**:
+- Understand the needs of the community
+- Have their interests aligned with the growth of the Algorand ecosystem
+ 
+### Template to submit a proposal
+
+``` json 
+{
+   "Title":"Name of the proposal",
+   "Project":"Name of the project",
+   "Details":"Details about the proposal",
+   "Category":"Submission Category, DEFI/NFT ...",
+   "Url":"Webiste url",
```

Comment:
I don't know how I feel about allowing proposers to include a URL. It's a potential attack vector and unfair to require xGovs visit anonymous external links.